### PR TITLE
Implement digital signing according to assignment persona

### LINKED ISSUE #401: Digital signing
**Replace the identification fields at the top of each form with a digital signing feature after data has been filled in.**
This is similar to the process that would happen in a professional setting, and would benefit students to learn before hitting a work environment. Included in this “signature” area should also be some kind of messaging that reminds the user that their signature is a confirmation that they have reviewed the entered information and that it is correct. 

Update June 2020.
Once https://github.com/BCcampus/edehr/issues/674 is complete use the assignment persona data to complete this digital signing feature. 

- remove the record header from displaying a the top of the forms (remove from Inputs)
- in the dialog footer add the following dynamic text and a checkbox. Disable the submit button until the checkbox is checked (default unchecked).
"Certified correct ${persona name}, ${persona profession}.  Hospital date: ${assignment date} ${assignment time in 24hr format}"

See comment below June 20th

### PR DESCRIPTION
**Pull request by viniciusarre**
Closes #401

### CODE REVIEW

File: client/src/outside/components/AssignmentsDialog.vue [hunk 1]
```diff
@@ -24,7 +24,26 @@
...
+                div(class="input-element input-element-full")
+                  label Profession
+                  input(class="input text-input", type="text", v-model="profession")
+              div(class="form-element")
+                div(class="input-element input-element-full")
+                  label Day
+                  input(class="input text-input", type="text", v-model="day")
+              div(class="form-element")
+                div(class="input-element input-element-full")
+                  label Time
```
[the remark this answers is not in the document]
Comment (bryan-gilbert) [reply client/src/outside/components/AssignmentsDialog.vue:44]: Also add validation Day must be a number >= 0. Time must be 24 hr format.
All fields are optional.  IF NOT DEFINED, then don't show in the EHR side.

File: client/src/inside/components/page/EhrElementCommon.vue [hunk 2]
```diff
@@ -128,6 +132,10 @@ export default {
         let initialValue = inputs[this.elementKey]
         if (dbDialog || dbInputs) console.log('EhrCommon key has value', this.key, initialValue)
         this.setInitialValue(initialValue)
+        // if (this.element.recHeader) {
```
Comment (bryan-gilbert) [inline client/src/inside/components/page/EhrElementCommon.vue:135]: Remove code if not needed.

File: client/src/inside/components/page/EhrElementCommon.vue [hunk 3]
```diff
@@ -57,6 +58,9 @@ export default {
     isEditing () {
       return this.ehrHelp.isEditing()
     },
+    personaData () {
```
Comment (bryan-gilbert) [inline client/src/inside/components/page/EhrElementCommon.vue:61]: rename to assignmentCaseStudyData and SH.getAssignmentCaseStudyData

File: client/src/inside/components/page/EhrDialogForm.vue [hunk 4]
```diff
@@ -1,16 +1,29 @@
...
+      @save="saveDialog", 
+      v-bind:errors="errorList", 
+      :hasLeftContent="true"
+      :disableSave="disableSave"
+    )
       h3(slot="header") {{ tableDef.addButtonText }}
       div tableKey {{ tableKey}}
       div(slot="body", class="ehr-page-content")
         ehr-group(v-for="group in groups", :key="group.gIndex", :group="group", :ehrHelp="ehrHelp")
+      div(slot="left-content", class="checkbox-wrapper", v-if="hasRecHeader")
```
Comment (bryan-gilbert) [inline client/src/inside/components/page/EhrDialogForm.vue:15]: Generally relabel the 'record header' to 'case study data'

**Comment by bryan-gilbert** [comment]
@viniciusarre You can, if you wish, merge this PR into your other PR and submit them as a new whole.

**Comment by viniciusarre** [comment]
Closing this, to fix the details and squash it all into a new PR, as discussed yesterday

**Pull request closed without merging**